examples/hitl/rearrange_v2/rearrange_v2.py

```diff
@@ -307,9 +395,24 @@ def on_enter(self):
                 user_index
             ].gui_agent_controller._agent_idx
 
+        episode = self._app_service.episode_helper.current_episode
+        self._session.session_recorder.start_episode(
+            episode.episode_id,
+            episode.scene_id,
+            episode.scene_dataset_config,
+            user_index_to_agent_index_map,
+        )
+
     def on_exit(self):
         super().on_exit()
 
+        episode_success = all(
```

Comment:
`episode_success` here means "user thinks episode was done/success", right?

Reply:
In the current state, success is the only outcome.

In a following PR, I'll be adding a way for users to report either success or failure via a form.